### DISCUSSION TITLE
How could I remove tsconfig charset

### DISCUSSION THREAD

**Discussion by helbing** [aws/jsii#4017] [General]
When I build with JSII, JSII will auto-generate the tsconfig file which contains the charset property. VSCode will report the error, Option 'charset' is deprecated and will stop functioning in TypeScript 5.5. How could I remove this property?

<img width="617" alt="image" src="https://user-images.githubusercontent.com/7275620/226252407-4e10e6b9-7e62-49c2-b833-146785412227.png">

**Comment by asssaver97** [comment]
Same problem, but the charset property is not generated in the zshrc environment.
